pubspec.yaml

```diff
@@ -1,11 +1,12 @@
 name: stripe
-version: 4.11.0
+version: 4.11.1
 description: A simple Stripe API wrapper, that is meant to be used on the server.
 homepage: https://github.com/enyo/stripe-dart
 documentation: https://github.com/enyo/stripe-dart/blob/main/README.md
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">3.0.0"
 dependencies:
+  http: ^1.2.0
```

Comment:
Do we need http here? We use dio in the package and it's probably not used at all.

Reply:
Fair point